First backbone

Usagib opened the pull request:
code review pull request for project: Displaying and inputting data

samgaco commented on index.html:
```diff
@@ -0,0 +1,96 @@
+	<div class="nav-bar">
+		<div>
+			<button class="sign-button">
+			<!--	<img class="s-logo" src="https://cdn4.iconfinder.com/data/icons/epic-outlines/30/667814-lock_4-512.png" alt="" /> -->
```
It is better to avoid commented code in your projects.

Usagib commented:
Hello Team!
we made the proper changes:
- We passed the validator test
- Corrected lint style errors
- Added some margins and background color to the form footer
- Added a LiveDemo using GitHub Pages!

Heading out the pull request for code review, Happy coding!
